debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/Filters.java

```diff
@@ -22,13 +22,21 @@
     protected static final Set<String> BUILT_IN_DB_NAMES = Collect.unmodifiableSet("local", "admin");
 
     private final Predicate<CollectionId> collectionFilter;
+    private final Predicate<String> databaseFilter;
 
     /**
      * Create an instance of the filters.
      * 
      * @param config the configuration; may not be null
      */
     public Filters(Configuration config) {
+        String dbWhitelist = config.getString(MongoDbConnectorConfig.DATABASE_WHITELIST);
+        if (dbWhitelist != null && !dbWhitelist.trim().isEmpty()){
+            databaseFilter = Predicates.includes(dbWhitelist);
+        } else {
+            databaseFilter = Predicates.includes(".*");
```

Comment:
Can we pass a simple predicate which always returns true instead of passing it through the regex engine?